openstack/baremetalintrospection/v1/introspection/results.go

```diff
@@ -0,0 +1,256 @@
+package introspection
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// introspectionResult ...
```

Comment:
I recommend removing this doc comment. `introspectionResult` is local to this package and not exposed to the user.